src/docs/self-hosted/releases.mdx

```diff
@@ -18,9 +18,30 @@ Self-hosted Sentry follows a monthly release schedule using the [CalVer](https:
 
 ## Upgrading
 
-We strongly recommend upgrading by going through all the versions between your current version and the target version you are upgrading to. That means if you are on version `20.6.0` and want to upgrade to `20.8.0`, the recommendation is for you to upgrade to `20.7.0` first and then upgrade to `20.8.0`. Skipping versions *may* work, but there will be times that Sentry requires specific a version to be installed first, to ensure essential data migrations before moving forward.
+We encourage everyone to regularly update their Sentry installations to get the best and the most recent Sentry experience.
 
-Upgrading is as simple as downloading the latest version of the onpremise repository and running `./install.sh` there. See the <Link to="/self-hosted/">main page for self-hosted Sentry</Link> if you need more information about upgrades and configuration.
+To upgrade, all you need to do is download or check out the version of onpremise repository you want, replace your existing folder's contents with that, and then run `./install.sh`. **We may have some updated configuration, especially for new features, so always check the example config files under the [sentry directory](https://github.com/getsentry/onpremise/blob/master/sentry/) and see if you need to update your existing configuration**. We do our best to automate critical configuration updates, but you should always check your configs during upgrades.
```

Comment:
Alert box of some kind instead of bold?

Reply:
Let's do that in a follow up as it is not really relevant to this PR?